README.md

```diff
@@ -95,6 +96,28 @@ Subsystem by running the following in PowerShell as an administrator
 Restart-Service LxssManager*
 ```
 
```

Comment:
I requested @kuthedk to place this section in the "Getting Started", but I just realized that the deployment part is documented after the "Testing & Static Analysis". 

Would it make more sense @kuthedk @JmScherer @fatimarabab  to move this new section @kuthedk wrote to be within the "Deployment" section and it go after the "Deploying with Docker-Compose" subsection.  The "Deployment" section can be renamed to be "Deploying and Using" or something similar.

Reply:
I would suggest moving it into the "Deployment" section since the login section assumes the user has already locally deployed.

> To log in to the system once locally deployed....

The "Deployment" section can be renamed. How about "Deployment and Usage"?

Reply:
Second 'Deployment and Usage', but good with any naming

Reply:
I changed it to `Deployment and Usage` as I liked that the best.